core/mvcc/database/mod.rs

```diff
@@ -242,6 +254,7 @@ impl<Clock: LogicalClock> MvStore<Clock> {
     /// * `row` - the row object containing the values to be inserted.
     ///
     pub fn insert(&self, tx_id: TxID, row: Row) -> Result<()> {
+        tracing::trace!("Insert -> TX: {}, Row: {:?}", tx_id, row.id);
```

Comment:
I would probably suggeest following a consistent log format. For example, I'm trying to do something like `insert(paramx=x, paramy=y)`